### PR TITLE
add: ovirt 4.4 repos

### PR DESCRIPTION
**Pull request by mnecas**
@mwperina @dangel101

### CODE REVIEW

**Review by mwperina — CHANGES_REQUESTED** [review]
4.4 GA files looks good to me,  but shouldn't we also remove 4.4 beta configuration? This will be part of RHV 4.4 GA, so it doesn't make any sense to install beta when GA is released

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Role Variables
 |--------------------------------------------|-----------------------|-------------------------------------------|
 | ovirt_repositories_ovirt_release_rpm       | UNDEF                 | URL of oVirt release package, which contains required repositories configuration. |
 | ovirt_repositories_use_subscription_manager| False                 | If true it will use repos from subscription manager and the value of <i>ovirt_repositories_ovirt_release_rpm</i> will be ignored. |
-| ovirt_repositories_ovirt_version           | 4.3                   | oVirt release version (Supported versions [4.1, 4.2, 4.3, 4.4-beta]). Will be used to enable the required repositories in case <i>ovirt_repositories_use_subscription_manager</i> is set. |
+| ovirt_repositories_ovirt_version           | 4.3                   | oVirt release version (Supported versions [4.1, 4.2, 4.3, 4.4]). Will be used to enable the required repositories in case <i>ovirt_repositories_use_subscription_manager</i> is set. |
```
Comment (mwperina) [inline README.md]: Please change the default to 4.4

**Review by mwperina — CHANGES_REQUESTED** [review]
It looks good to me, just please switch the default to 4.4

**Comment by mwperina** [comment]
@dangel101

**Review by sandrobonazzola — APPROVED** [review]
Looks good to me

**Pull request merged**